config/environments/production.rb

```diff
@@ -56,21 +57,23 @@
   # Send deprecation notices to registered listeners
   config.active_support.deprecation = :notify
 
-  config.logger = ActiveSupport::TaggedLogging.new(Logger.new(STDOUT))
-  config.propub_url_root = "/politwoops"
-  config.assets.compress = true
+  #config.logger = ActiveSupport::TaggedLogging.new(Logger.new(STDOUT))
+  #config.propub_url_root = "/politwoops"
+
+  config.assets.compress =true
   config.assets.compile = true
   config.assets.digest = true
 
   config.paperclip_defaults = {
     :storage => :s3,
-    :s3_permissions => :public_read,
     :path => "/:attachment/:filename",
-    :url => "https://s3.amazonaws.com/pp-projects-static/",
+    :url => ":s3_domain_url",
+    :s3_domain_url => "http://cfa-politwoops-ke.s3.amazonaws.com/",
```

Comment:
@chegejames Is this correct? I fill like there should be region involved somewhere.